plugin/plugin.go

```diff
@@ -220,7 +228,10 @@ func (p *Plugin) Init(ctx context.Context, spec []byte, options NewClientOptions
 
 	options.PluginMeta = p.Meta()
 
-	p.client, err = p.newClient(ctx, p.logger, spec, options)
```

Comment:
Do we want to add this field to the log only if it's not empty?

Reply:
https://github.com/cloudquery/cloudquery/issues/17322